elements/rh-announcement/test/rh-announcement.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, html } from '@open-wc/testing';
+import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+import { RhAnnouncement } from '@rhds/elements/rh-announcement/rh-announcement.js';
+
+describe('<rh-announcement>', function() {
+  describe('simply instantiating', function() {
+    let element: RhAnnouncement;
+    it('imperatively instantiates', function() {
+      expect(document.createElement('rh-announcement')).to.be.an.instanceof(RhAnnouncement);
+    });
+
+    it('should upgrade', async function() {
+      element = await createFixture<RhAnnouncement>(html`<rh-announcement></rh-announcement>`);
+      const klass = customElements.get('rh-announcement');
+      expect(element)
+        .to.be.an.instanceOf(klass)
+        .and
+        .to.be.an.instanceOf(RhAnnouncement);
+    });
```

Comment:
Definitely going to want to add a few more tests here. I would look into adding a test that includes the a11y snapshot. `rh-switch` has some [a11y snapshot tests](https://github.com/RedHat-UX/red-hat-design-system/blob/staging/charmander/elements/rh-switch/test/rh-switch.spec.ts#L4).

Reply:
Added an a11y test in https://github.com/RedHat-UX/red-hat-design-system/pull/1722/commits/d10a6887bd4f8aa87955b23b70ee362607617536. Should we add any other tests?